models/paper2/stmts_vs_pmids.py

```diff
@@ -0,0 +1,92 @@
+import pickle
```

Comment:
I think this file may have been added by accident, we could remove it from the PR.